omise-wp-admin.php

```diff
@@ -204,7 +233,36 @@ function register_dashboard_script() {
 			wp_enqueue_script( 'omise-dashboard-js', plugins_url( '/assets/javascripts/omise-dashboard-handler.js', __FILE__ ), array( 'jquery' ), OMISE_WOOCOMMERCE_PLUGIN_VERSION, true );
 			wp_enqueue_style( 'omise-css', plugins_url( '/assets/css/omise-css.css', __FILE__ ), array(), OMISE_WOOCOMMERCE_PLUGIN_VERSION );
 		}
+
+		function list_charges() {
```

Comment:
@nimid maybe we can move this method to `/includes/classes/class-omise-charge.php` and use it like,

```php
public function init_dashboard() {
    $viewData = $this->init();
 
    try {
        $viewData['charges'] = Omise_Charge::list( $params );
        
        $this->render_view( 'includes/templates/omise-wp-admin-page.php', $viewData );
    } catch( Exception $e ) {
        echo "<div class='wrap'><div class='error'>" . esc_html( $e->getMessage () ) . "</div></div>";
    }
}
```

what do you think?